cmd/syft/internal/commands/attest.go

```diff
@@ -247,7 +247,7 @@ func predicateType(outputName string) string {
 }
 
 func generateSBOMForAttestation(ctx context.Context, id clio.Identification, opts *options.Catalog, userInput string) (*sbom.SBOM, error) {
-	src, err := getSource(opts, userInput, onlyContainerImages)
+	src, err := getSource(ctx, opts, userInput, stereoscopesource.ImageTag)
```

Comment:
I don't think this is the right thing to do -- aren't there only very specific sources needed here? not just that we got a container imager, but the source needs to be a registry perhaps so cosign can upload an attestation?